packages/app/app/containers/HelpModalContainer/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import HelpModal from '../../components/HelpModal';
+import { ContributorProps } from '../../components/HelpModal/Contributors';
+import { githubContribSelectors } from '../../selectors/githubContrib';
+
+const HelpModalContainer: React.FC<{}> = () => {
+
+  const githubContrib: ContributorProps = {
+    contributors: useSelector(githubContribSelectors.contributors),
+    loading: useSelector(githubContribSelectors.loading),
+    error: useSelector(githubContribSelectors.error)
+  };
+
+  return (<HelpModal githubContrib={githubContrib} />);
```

Comment:
It might be useful to move the callbacks opening links and usestate/usecallback calls here as well.
We generally want hook calls to happen in containers. The exception is `useTranslation`, which can be placed in components, unless they're in the `ui` package - those should take strings as props to show text.